core/templates/dev/head/pages/exploration_editor/simple_editor_tab/SimpleEditorSidebarDirective.js

```diff
@@ -25,6 +25,8 @@ oppia.directive('simpleEditorSidebar', [function() {
         'ScrollSyncService', 'QuestionIdService',
         function($scope, EditorModeService, SimpleEditorManagerService,
                  ScrollSyncService, QuestionIdService) {
+          
+          Ps.initialize(document.querySelector('simple-editor-sidebar'));
```

Comment:
For future reference, please add a brief comment above this line explaining what the point of this is (and why we need the Ps dependency in the first place).